content/pages/education/apply-for-education-benefits/application.md

```diff
@@ -4,6 +4,14 @@ entryname: edu-benefits
 layout: page-react.html
 ---
 <div id="main">
+  <nav class="va-nav-breadcrumbs">
+    <ul class="row va-nav-breadcrumbs-list" role="menubar" aria-label="Primary">
+      <li><a href="/">Home</a></li>
+      <li><a href="/education/">Education</a></li>
+      <li class="parent"><a href="/education/apply-for-education-benefits/">Apply for Education Benefits</a></li>
```

Comment:
I think it's good as is. Seems like a small thing to have to remember to change later.